src/DependencyInjection/DependencyAware.class.js

```diff
@@ -19,4 +19,13 @@ export default class DependencyAwareClass {
   getContainer() {
     return this.di;
   }
+
+  /**
+   * Shortcut for `this.getContainer().definitions`
+   *
+   * @returns {object}
+   */
+  get definitions() {
```

Comment:
Wondering if we should continue with `getDefinitions()` style methods rather than `get definitions()` etc, just for the sake of internal consistency within lambda wrapper?

Although as you know I'm not too bothered by this sort of thing so happy not to

Reply:
I love consistency, but I'd rather change `getContainer` into `get container` and have a more expressive API. @seb-cr what do you think? Should we rename those methods to `getDefinitions()` or just leave it for now waiting for the breaking change of renaming `getContainer`?

Reply:
I really don't think the breaking change of renaming `getContainer` is worth it! IMO breaking changes in this repo should be kept to a minimum

Reply:
It's a good question. I prefer the getter to a `getDefinitions` method, and would vote to stick with it in the interest of writing new code in a preferred style. I don't really mind either way though!

Reply:
Yes I agree @tomisaacmarshall but we are going to have a massive breaking change once we migrate to TS and we will leave the lose typing of `DependencyInjection`, so that will be the natural moment to rename those methods.

Reply:
Ok